src/components/previewers/VideoPlayer.jsx

```
@@ -0,0 +1,170 @@
+import React, { useCallback, useMemo, useState } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)